quantum/keymap_extras/keymap_slovenian.h

```diff
@@ -137,11 +137,11 @@
 #define SI_CARN ALGR(SI_2)    // ˇ (dead)
 #define SI_CIRC ALGR(SI_3)    // ^ (dead)
 #define SI_BREV ALGR(SI_4)    // ˘ (dead)
-#define SI_DEG  ALGR(SI_5)    // ° (dead)
+#define SI_RNGA ALGR(SI_5)    // ° (dead)
```

Comment:
Interesting... the Slovenian layout produces the degree symbol if you cancel the dead key; it is indeed technically a "ring above" symbol due to its deadness... but seems to only combine with "u".

Reply:
That's correct. Most layouts that have a ring above dead key actually use the degree sign, just flagged for deadness.

Another one that threw me off is the low single quote dead key in the Serbian layout (`RS_LSQU`), which behaves as a cedilla (`c`→`ç`, `s`→`ş`), but when you cancel it you do get the low single quote character. I opted not to change that one, though, since there already is an actual cedilla key in that layout...

Reply:
I think the `RNGA`s in the Serbian Latin and Croatian layouts also only combine with U to give Ů (used in Czech).